docs/marketplace/user/merchant-portal-user-guides/202108.0/offers/managing-product-offers.md

```diff
@@ -16,108 +16,60 @@ Review the reference information in this article before you start, or just look
 
 To create a product offer:
 
-1. Click **Add Offer**.
-2. From the list of concrete products, select the necessary product you want to create an offer for, hover over the three dots, or just click the line and then click **Create Offer**.
-3. In the drawer, fill the necessary fields and click **Create**.
+1. Click **+Add Offer**.
+2. On the _Create Offer_ page, in the list of concrete products, click on the product you want to create an offer for, or hover over the three dots and then click **Create Offer**. The drawer opens.
+3. In the _Offer status_ pane, select the **Offers is Active** check box to make the product offer visible on the Storefront. If you don't want the product offer to be displayed on the Storefront, skip this step.
+4. In the _Merchant SKU_ pane, enter an SKU.
+5. In the _Stores_ pane, from the drop-down list, select stores for which the offer is available.
+6. In the _Available Stock_ pane, set the product offer quantity:
+    1. In the **QUANTITY** field, enter the quantity of the product currently available in stock.
+    2. (Optional) Select **Always in Stock** to make the product offer always available in terms of stock. Otherwise, when the number of orders exceeds the number of products set in the **QUANTITY** field, the product offer becomes out-of-stock on the Storefront.
```

Comment:
```suggestion
    2. Optional: Select **Always in Stock** to make the product offer always available in terms of stock. Otherwise, when the number of orders exceeds the number of products set in the **QUANTITY** field, the product offer becomes out-of-stock on the Storefront.
```